mycore-pi/src/main/java/org/mycore/pi/MCRPIRegistrationService.java

```diff
@@ -43,7 +43,7 @@
 
     public static final String GENERATOR_CONFIG_PREFIX = "MCR.PI.Generator.";
 
-    public static final String METADATA_MANAGER_CONFIG_PREFIX = "MCR.PI.Inscriber.";
+    public static final String METADATA_MANAGER_CONFIG_PREFIX = "MCR.PI.MetadataManager.";
```

Comment:
Is this also part of deprecated properties? We should prevent property renaming within an LTS version ...

Reply:
It was renamed pre LTS but i forgot to rename this constant. See also:      
public static final String METADATA_MANAGER_DEPRECATED_CONFIG_PREFIX = "MCR.PI.Inscriber.";

But that does not matter because everything is backward compatible (You just get a log message).